scripts/check_files.py

```diff
@@ -374,7 +374,7 @@ class LicenseIssueTracker(LineIssueTracker):
     if build_tree.is_mbedtls_3_6():
         LICENSE_EXEMPTION_RE_LIST.append(r'3rdparty/(?!(p256-m)/.*)')
     else:
-        LICENSE_EXEMPTION_RE_LIST.append(r'tf-psa-crypto/drivers/(?=(everest)/.*)')
+        LICENSE_EXEMPTION_RE_LIST.append(r'(tf-psa-crypto/)?drivers/(?=(everest)/.*)')
```

Comment:
For things to be more explicit, I'd say rather something like (not tested):
```python
if build_tree.looks_like_tf_psa_crypto_root(os.getcwd()):
    LICENSE_EXEMPTION_RE_LIST.append(r'drivers/(?=(everest)/.*)')
elif build_tree.is_mbedtls_3_6():
    LICENSE_EXEMPTION_RE_LIST.append(r'3rdparty/(?!(p256-m)/.*)')
```
The script uses `git ls-files` to get the list of files to check and this does not cover submodules. Thus now in mbedtls:development files under `tf-psa-crypto/drivers` are not checked by the script thus no need to update LICENSE_EXEMPTION_RE_LIST in that case.

Reply:
Updated, thanks.